### PR TITLE
Create image nginx with FIO

### PR DESCRIPTION
**Pull request by OdedViner**
In our tests, we create an NGINX pod and then Install FIO [via `oc rsh` cmd]
Created a `nginx:alpine` image with FIO and build it with multi-architecture support. 
Now, we do not need to install FIO on the existing pod [save running time]

1. Install Docker Buildx if you haven't already:
```
docker buildx version  # Check if it's already installed
docker buildx create --use
```

2. Create a Dockerfile for NGINX:
```
# Use a base image with QEMU for multi-architecture support
FROM --platform=$BUILDPLATFORM nginx:alpine

RUN apk update 
RUN apk upgrade
RUN apk add fio bash
```
3. Build and Push the multi-architecture Docker image using Docker Buildx:
```
 docker buildx build --platform linux/amd64,linux/ppc64le,linux/s390x,linux/arm64 -t quay.io/ocsci/nginx:fio . --push
```

I saved the image on the same registry and changed the tag to `fio`
https://quay.io/repository/ocsci/nginx?tab=tags

### CODE REVIEW

File: ocs_ci/ocs/workload.py [hunk 1]
```diff
@@ -47,23 +47,23 @@ def __init__(
...
-                workload to workload. Refer constants.TEMPLATE_WORKLOAD_DIR
-                for various available workloads
-
-        Returns:
-            bool: True if setup is success else False
-        """
-        if self.pod:
-            setup_conf["pod"] = self.pod
-        return self.work_load_mod.setup(**setup_conf)
+    # def setup(self, **setup_conf):
```
Comment (jilju) [inline ocs_ci/ocs/workload.py:50]: Won't this affect the non-nginx pods ?

Comment (OdedViner) [reply ocs_ci/ocs/workload.py:50]: @shylesh What do you think? Do we use Debian only for NGINX.. correct?

Comment (shylesh) [reply ocs_ci/ocs/workload.py:50]: @OdedViner I see another reference https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/pod.py#L541 which is jenkins workload

**Review by petr-balogh — APPROVED** [review]

**Review by shylesh — APPROVED** [review]
LGTM

**Comment by abdulkandathil** [comment]
@OdedViner The new tag is not compatible with s390x architecture.. could you please rebuild the image..

```
Events:
  Type     Reason                  Age               From                     Message
  ----     ------                  ----              ----                     -------
  Normal   Scheduled               24s               default-scheduler        Successfully assigned namespace-test-b65c47a619054820aebf22c9d/pod-test-rbd-0ff1cfb74f8e46baa38dfde8d8c to worker-1.ocsm1301001.lnxero1.boe
  Normal   SuccessfulAttachVolume  24s               attachdetach-controller  AttachVolume.Attach succeeded for volume "pvc-45553ae9-1d7e-48e8-bff0-964eba77420c"
  Normal   AddedInterface          20s               multus                   Add eth0 [10.131.0.110/23] from openshift-sdn
  Normal   Pulled                  4s (x3 over 20s)  kubelet                  Container image "quay.io/ocsci/nginx:fio" already present on machine
  Normal   Created                 4s (x3 over 20s)  kubelet                  Created container web-server
  Normal   Started                 4s (x3 over 20s)  kubelet                  Started container web-server
  Warning  BackOff                 3s (x3 over 18s)  kubelet                  Back-off restarting failed container web-server in pod pod-test-rbd-0ff1cfb74f8e46baa38dfde8d8c_namespace-test-b65c47a619054820aebf22c9d(d2b4ffb9-da47-4c1c-a0cb-e6ec6ef0c1ee)
[root@m1301001 ~]# oc -n namespace-test-b65c47a619054820aebf22c9d logs pod-test-rbd-0ff1cfb74f8e46baa38dfde8d8c
exec /docker-entrypoint.sh: exec format error
[root@m1301001 ~]#
```

**Pull request merged**